### DISCUSSION TITLE
Time out when "send to device"

### DISCUSSION THREAD

**Discussion by toreriksolem** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#66] [Editor App Feature Requests] [answered]
If I impatiently alt+tab to another program when having pressed "send to device" within the app, it seems that the sync-operation times out.

For now I am resolving this by refreshing the browser and power cycling the Bridge.

MacBook
 OS 11.5.1 
Running web app in Chrome Version 104.0.5112.79
USB-C cable directly to Bridge 6

**Comment by konnsim** [comment, marked as answer]
Duplicate of Import/Export success rate is very unstable #69.
